frontend/rolecall/src/app/common_components/full-name.pipe.ts

```diff
@@ -0,0 +1,19 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {User} from '../api/user_api.service';
+
+@Pipe({
+  name: 'fullName'
+})
+export class FullNamePipe implements PipeTransform {
+
+  public transform(user: User, ...args: unknown[]): string {
```

Comment:
This pipe is not accepting arguments, so remove `...args: unknown[]` and just leave `user: User`

Reply:
Done.